lib/orbit/import/rfid.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Orbit.Import.Rfid do
+  require Logger
+  alias Orbit.Employee
+  alias Orbit.Repo
+  alias Orbit.S3
+
+  @spec download() :: binary()
+  def download do
+    {:ok, csv_data} =
+      S3.read(
+        :glides_global,
+        Keyword.fetch!(Application.fetch_env!(:orbit, Orbit.Import.Rfid), :pathname)
+      )
+
+    csv_data
+  end
+
+  @badge_number_field "EMPLOYEE_ID"
+  @badge_serial_field "SERIAL_NUMBER"
+
+  @spec import_rows(Enumerable.t()) :: :ok
```

Comment:
Since we know what kind of Enumerable this is, can we specify?

Reply:
The typespec of `CSV.decode!` doesn't get more specific than `Enumerable.t()`, so I think this is the strongest we can get without making additional assumptions about the file format that should be handled at runtime rather than via types.